inc/classes/class_plugin.php

```diff
@@ -1,28 +1,41 @@
 <?php
 
 /**
- * Class PLUGIN for Lansuite
+ * Class plugin
  *
- * This Class generates the needed Dataarray to load the Pluginfiles
- *
- * @package lansuite_core
- * @author bytekilla
```

Comment:
Is there a good reason to remove code attribution to @byte666?

Reply:
In my opinion the @author-tags are better filled by a `git blame` or a `git log`.
The history of LanSuite is part of this repository.
The software is developed by a team and it is a team effort.
If i want to know who wrote it, i will fire a `git blame`.
I am thankful for every line that was written by someone here.

The sentene `This Class generates the needed Dataarray to load the Pluginfiles` was removed, because i have to admit: For me it says nothing useful. What Dataarray?
The context is missing IMO.

Reply:
OK, fine by me ;)